src/__tests__/RepoPage/Repo.test.js

```diff
@@ -82,6 +82,6 @@ describe('Repo details component', () => {
     jest.spyOn(api, 'get').mockResolvedValue({ status: 200, data: { data: mockRepoDetailsData } });
     render(<RepoDetails />);
     fireEvent.click(await screen.findByTestId('pullcopy-btn'));
-    await waitFor(() => expect(mockCopyToClipboard).toHaveBeenCalledWith('Pull test'));
+    await waitFor(() => expect(mockCopyToClipboard).toHaveBeenCalledWith('docker pull http://localhost/test'));
```

Comment:
This needs to have multiple tabs, and the user can select docker/podman/skopeo.

For docker: `docker pull 127.0.0.1:5000/alpine:latest`
For podman: `podman pull 127.0.0.1:5000/alpine:latest`
For skopeo: `skopeo copy docker://127.0.0.1:5000/alpine:latest docker://artprod.mycompany:port/<DOCKER_REPOSITORY>:<DOCKER_TAG>`

Where `alpine`, `latest` are information on this specific repo/tag (this feature should be available after you select an image:tag). Also the source where you copy from should not be hardcoded, you can probably get the hostname from some configuration (it's the same hostname where the backend resides).

Reply:
This is the test file, the logic is (currently) in the RepoDetails.jsx file

Reply:
Ok, I see. nevermind then :)